tests/python/utils_binding.py

```diff
@@ -170,14 +168,14 @@ def initXRT(opt):
             if (ip[i].m_type != 1):
                 continue
             opt.cu_base_addr = ip[i].m_base_address
-            print("CU[%d] %s @0x%x") % (i, ctypes.cast(ip[i].m_name, ctypes.c_char_p).value, opt.cu_base_addr)
+            print("CU[%d] %s @0x%x") % (i, ctypes.cast(ip[i].m_name, ctypes.c_char_p), opt.cu_base_addr)
```

Comment:
Dropping the **.value** confuses print. I see the following for 22_verify testcase:

`CU[0] c_char_p(139712362987648) @0x1800000`

instead of

`CU[0] hello:hello_1 @0x1800000 `